aiida/cmdline/commands/cmd_run.py

```diff
@@ -114,7 +114,7 @@ def run(scriptname, varargs, auto_group, auto_group_label_prefix, exclude, inclu
 
     try:
         # Here we use a standard open and not open, as exec will later fail if passed a unicode type string.
-        handle = open(scriptname, 'r')
+        handle = open(scriptname, 'r')  # pylint: disable=consider-using-with
```

Comment:
Idem: instead of disabling, couldn't we actually not better use a with statement here?

Reply:
as above; basically I would maybe open a separate issue to look at removing all the `disable=consider-using-with`, because some of them feel like they need a bit of scrutiny to double check they won't break the current behaviour

Reply:
Good idea to open an issue and leave all of these with disable statements for now